.ci/Jenkinsfile

```diff
@@ -143,6 +143,34 @@ pipeline {
         }
       }
     }
+    stage('BDD Tests') {
+      when {
+        beforeAgent true
+        allOf {
+          expression { return env.ONLY_DOCS == "false" }
+          anyOf {
+            not { changeRequest() }
+            expression { return params.Run_As_Master_Branch }
+          }
+        }
+      }
+      options { skipDefaultCheckout() }
+      environment {
+        HOME = "${env.WORKSPACE}"
+        PATH = "${env.WORKSPACE}/bin:${env.PATH}"
+      }
+      steps {
+        withGithubNotify(context: 'BDD Tests') {
+          deleteDir()
+          unstash 'source'
+          dir(BASE_DIR) {
+            catchError(buildResult: 'SUCCESS', stageResult: 'UNSTABLE', message: 'BDD Tests failed but keep running the build') {
+              sh(label: "Run BDD tests", script: "./spec/scripts/features.sh ruby:2.7")
```

Comment:
As of now, it only runs for one Ruby version. Should it be run for all of them?

Reply:
Sure 👍

Reply:
Then we can integrate it into the main matrix

Reply:
I added 5880d66c5b1acbbaac736fb9b174c2832fe0f230 to run the BDD tests with every Ruby version, for each CI build. Do we want to run them on PRs too?